packages/jellyfish-api-core/__tests__/category/blockchain/getMempoolAncestors.test.ts

```diff
@@ -0,0 +1,73 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import BigNumber from 'bignumber.js'
+import { MempoolTx } from '../../../src/category/blockchain'
+import waitForExpect from 'wait-for-expect'
+import { Testing } from '@defichain/jellyfish-testing'
+
+describe('getMempoolAncestors', () => {
+  const container = new MasterNodeRegTestContainer()
+  const testing = Testing.create(container)
+
+  beforeAll(async () => {
+    await testing.container.start()
+    await testing.container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await testing.container.stop()
+  })
+
+  it('should return no ancestor ids for tx without ancestors', async () => {
+    const txId = await testing.rpc.wallet.sendToAddress('mwsZw8nF7pKxWH8eoKL9tPxTpaFkz7QeLU', 0.0003)
+    const mempoolEntry: string[] = await testing.rpc.blockchain.getMempoolAncestors(txId, false)
+    await waitForExpect(async () => {
```

Comment:
Is there a need for a `waitForExpect` here? I'm pretty sure this test will pass without it.